manifests/0000_30_cluster-api_00_credentials-request.yaml

```diff
@@ -70,14 +70,77 @@ metadata:
     include.release.openshift.io/self-managed-high-availability: "true"
     release.openshift.io/feature-set: "TechPreviewNoUpgrade"
 spec:
```

Comment:
@abutcher  Do we now need to set a custom `.spec.cloudTokenPath` to `/var/run/secrets/azure/tokens` given
https://github.com/openshift/cluster-capi-operator/pull/124/files#r1273768474 ?

cc. @odvarkadaniel

Reply:
Yes, I believe so.

Reply:
Thanks for the guidane here @jstuever.
@odvarkadaniel would you be able to add that? If not let me know and I can take this over

Reply:
Added.